### PR TITLE
Allow configurable visibility

### PR DESCRIPTION
**Pull request by ghost**
## What does this PR do?
Allows visibility of the view to be configurable
#### Normal
```jsx
import React from "react";
import { TypingAnimation } from 'react-native-typing-animation';

class Example extends React.Component {
  render() {
    return (
      <TypingAnimation 
        dotColor="black"
        dotMargin={10}
        dotAmplitude={3}
        dotSpeed={0.08}
        dotRadius={6.5}
        dotX={12}
        dotY={6}
      />
    );
  }
}
```
#### Output
normal

##
#### Hidden
```jsx
import React from "react";
import { TypingAnimation } from 'react-native-typing-animation';

class Example extends React.Component {
  render() {
    return (
      <TypingAnimation 
        dotColor="black"
        dotMargin={10}
        dotAmplitude={3}
        dotSpeed={0.15} // Can be left out
        dotRadius={6.5}
        dotX={12}
        dotY={6}
        show={false}
      />
    );
  }
}
```
#### Output
blank

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -31,8 +31,8 @@ class TypingAnimation extends React.Component {
   }
 
   render() {
-    const { dotStyles, style, dotColor, dotMargin, dotRadius, dotX } = this.props;
-
+    const { dotStyles, style, show, dotColor, dotMargin, dotRadius, dotX } = this.props;
+    show ? style.visibility = "visible" : style.visibility = "hidden";
```
Comment (watadarkstar) [inline src/index.js]: This mutation seems off to me @murageden. Mutating styles in the `render` function feels like an anti-pattern. 

Can we make this more functional and instead do this:

```suggestion
    if(!show) return null;
```

Comment (ghost) [reply src/index.js]: My thinking is we still want to keep the space occupied by the view such that the view doesn't appear to be pushing itself in and out. Following your suggestion, the space is likely to become a bit condensed, let me test it and see, though.

Comment (watadarkstar) [reply src/index.js]: @murageden Could you send me a [loom](https://www.loom.com/) video of what you are seeing when you get a chance? :)

Comment (ghost) [reply src/index.js]: You're right Adrian, this implementation is [anti-pattern](https://daveceddia.com/why-not-modify-react-state-directly/). And I feel like this feature will become redundant. The `style` prop should be used instead, to achieve the intended functionality like so: 
```jsx
import React from "react";
import { TypingAnimation } from 'react-native-typing-animation';

class Example extends React.Component {
  render() {
    return (
      <TypingAnimation 
        dotColor="black"
        dotMargin={10}
        dotAmplitude={3}
        dotSpeed={0.08}
        dotRadius={6.5}
        dotX={12}
        dotY={6}
        style={{visibility: "hidden"}}
      />
    );
  }
}
```
This is easier and more consistent with the design. What do you think?

Comment (ghost) [reply src/index.js]: I made the changes tho.

**Review by watadarkstar — APPROVED** [review]
@murageden LGTM! I will merge this in and release it at a later point 👍

File: src/index.js (hunk 1 above)
Comment (watadarkstar) [reply src/index.js]: @murageden I think it is ok to have this extra `show` prop for extra convenience. Thank you for contributing 😄

Comment (dharmendrashah) [reply src/index.js]: if its work no other change requre

**Pull request merged**